modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -1198,13 +1198,19 @@ Maximum number of bytes returned in a fetch request.
 
 The strategy used to fulfill fetch requests.
 
+* `polling`: 
+
+* `non_polling`: The backend is signaled when a partition has new data, so it doesn’t need to repeatedly read from every partition in the fetch. This should improve fetch latency and CPU utilization for most workloads.
+
+* `non_polling_with_debounce`: Allows for fetches to be delayed before they begin to work. This results in fewer/larger fetch responses, reducing overall reactor utilization, and allowing for other requests such as produces to increase. This option will add a fixed delay as specified by <<fetch_reads_debounce_timeout,`fetch_reads_debounce_timeout`>> to the start of every fetch.
```

Comment:
Should we add a quick comment that it behaves the same as the `non_polling` option in addition to the extra functionality  allowing for delays?

Reply:
Included your note and improved readability.